### DISCUSSION TITLE
KPFCNN model on XYZRGB-Intensity

### DISCUSSION THREAD

**Discussion by Yarroudh** [WeikaiTan/Toronto-3D#16] [General]
I've trained KPFCNN model on [Toronto3D](https://github.com/WeikaiTan/Toronto-3D) Mobile LiDAR dataset using **XYZRGB-Intensity**. The results are presented as follow:

**Overall Accuracy:** 94.7%
**mean Intersection-over-Union:** 79%
**IoU of each class (ignored unclassified):**

1. Road: 96.6%
2. Road markings: 61.7%
3. Natural: 95.1%
4. Building: 89.1%
5. Utility line: 82.5%
6. Pole: 78.4%
7. Car: 87.1%
8. Fence: 41.8%

For comparison, the results you've reported using coordinates only:

OA: 95.39%
mIoU: 69.11%

![image](https://github.com/HuguesTHOMAS/KPConv-PyTorch/assets/72500344/73c14c4b-7f1f-48d0-8100-34408dcea7ee)

I've created a [pull request](https://github.com/HuguesTHOMAS/KPConv-PyTorch/pull/227) to KPConv-PyTorch repository and I can also provide the pretrained model if needed.
